setup.py

```diff
@@ -64,7 +64,7 @@
     "nncf": ["nncf>=2.11.0"],
     "openvino": ["nncf>=2.11.0", "openvino==2024.5.0", "openvino-tokenizers==2024.5.0"],
     "neural-compressor": ["neural-compressor[pt]>3.0", "accelerate", "transformers<4.46"],
-    "ipex": ["intel-extension-for-pytorch", "transformers>=4.39,<4.45"],
+    "ipex": ["intel-extension-for-pytorch", "transformers>=4.39,<4.47"],
```

Comment:
if my understanding is correct
```suggestion
    "ipex": ["intel-extension-for-pytorch>=2.4", "transformers==4.46"],
```

Reply:
Fixed, we have 2 versions of transformers in tests: [4.46.0, 4.46.3]